frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/top-consumers-card/utils.ts

```diff
@@ -19,6 +19,25 @@ export const sortResources: SortResourcesProps = (a, b) => {
   const bVal = _.get(b, 'values');
   const x = _.get(a, ['values', aVal.length - 1, 1]);
   const y = _.get(b, ['values', bVal.length - 1, 1]);
+  if (x === y) {
+    const aNameVal = _.get(a, 'metric');
```

Comment:
We have metric type available with us[1].

Pass metric type and use it.

```suggestion
topConsumerStatsResult.sort(sortResources(metricType))
```
[1]. https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/top-consumers-card/top-consumers-card-body.tsx#L48

Reply:
@cloudbehl passing any additional argument to compare function inside `Array.prototype.sort()` can lead to a complicated code. I have passed it as in a normal function.